scrapy_streaming/communication/map.py

```diff
@@ -1,24 +1,31 @@
-import json
+import base64
+
+import simplejson as json
 
 from scrapy.utils.python import to_unicode, to_native_str
 
-from scrapy_streaming.communication import wrappers
-from scrapy_streaming.utils import MessageError
+from scrapy_streaming.communication import validators
+from scrapy_streaming.utils import MessageError, extract_instance_fields
 
 
 class CommunicationMap(object):
     """
-    Helper class to create the json messages
+    Helper class to create and receive json messages
     """
 
     mapping = {
-        'spider': wrappers.SpiderMessage,
-        'request': wrappers.RequestMessage,
-        'log': wrappers.LogMessage
+        'spider': validators.SpiderMessage,
+        'request': validators.RequestMessage,
+        'form_request': validators.FormRequestMessage,
```

Comment:
@eLRuLL 
Yeah, from_response_request is really a better name. I renamed it in the last commit.
Also, I added a new test spider (https://github.com/aron-bordin/scrapy-streaming/blob/bfae10172f4f8a08b44a6339a8a8db51c7eaa890/tests/spiders/multiple_data.py) that sends a long message with 1000 log messages to check if the scrapy-streaming buffering is getting everything.